bridge/src/index.ts

```diff
@@ -296,6 +297,22 @@ process.on("uncaughtException", console.error);
         "string"
     );
 
+    const PLANET_ODIN_ID: string | undefined = Configuration.get(
+        "PLANET_ODIN_ID",
+        true,
+        "string"
+    );
+    const PLANET_HEIMDAL_ID: string | undefined = Configuration.get(
```

Comment:
need to modify spelling: `HEIMDAL` -> `HEIMDALL`

Reply:
Fixed Typos